app/Http/Controllers/CommentsController.php

```diff
@@ -138,13 +140,13 @@ public function delete(Request $request, $comment_id)
             $comment->delete();
             return response()->json([
                 'data' => ['comment' => $comment_id],
-                'message' => 'Comment deleted successfully',
+                'data' => 'Comment deleted successfully',
                 'response' => 'Ok'
```

Comment:
Why are you removing message returned and assigning the message to data key in the array ?